ample/util/ample_util.py

```diff
@@ -87,36 +89,55 @@ def extract_and_validate_models(amoptd, sequence=None, single=True, allsame=True
        only extract a file if the suffix is in the list
 
     """
+    
+    def is_tar_archive(filename):
```

Comment:
I think you might be able to use `tarfile.is_tarfile()` to do this

Reply:
Nice! There's a similar function in zipfile too so I'lll use that as well.